application/eventbridge_dlq_handler/eventbridge_dlq_handler.py

```diff
@@ -22,16 +22,24 @@ def lambda_handler(event: SQSEvent, context: LambdaContext) -> None:
         context (LambdaContext): Lambda function context object
     """
     record = next(event.records)
+    attributes = record.message_attributes
     message = record.body
     body = extract_body(message)
     logger.set_correlation_id(body["correlation_id"])
     logger.append_keys(dynamo_record_id=body["dynamo_record_id"])
     logger.append_keys(message_received=body["message_received"])
     logger.append_keys(ods_code=body["ods_code"])
+    error_msg = attributes["ERROR_MESSAGE"]["stringValue"]
+    attributes_error_msg_http_codes = [int(str) for str in error_msg.split() if str.isdigit()]
     logger.warning(
         "Eventbridge Dead Letter Queue Handler received event",
         extra={
             "report_key": DLQ_HANDLER_REPORT_ID,
-            "dlq_event": body,
+            "dlq_event_attributes_error_msg": attributes["ERROR_MESSAGE"]["stringValue"],
+            "dlq_event_attributes_error_msg_http_code": attributes_error_msg_http_codes[0],
+            "dlq_event_attributes_error_code": attributes["ERROR_CODE"]["stringValue"],
```

Comment:
are all these attributes always present or is there a chance this could error because the attribute is missing/